package.json

```diff
@@ -3,12 +3,12 @@
     "version": "1.0.4",
     "private": true,
     "scripts": {
-        "prestart": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner",
-        "buildnavplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && (yarn unlink > /dev/null || true) && yarn link",
+        "prestart": "yarn run buildnavplugin && yarn run buildcookiesplugin",
+        "buildnavplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && (yarn unlink > /dev/nul || true) && yarn link",
```

Comment:
This should be `/dev/null`.

Reply:
Oh sorry, I work on Windows PC so I was testing the scripts with just "nul" 🤦‍♂️

Reply:
We should be good to go now, sorry for all the back and forth.

Reply:
No worries ;)